src/bots/xai/GrokAPIBot.js

```
@@ -0,0 +1,10 @@
+import xAIAPIBot from "./xAIAPIBot";
+
+export default class GrokAPIBot extends xAIAPIBot {
+  static _className = "GrokAPIBot";
+  static _logoFilename = "grok-logo.svg";
+  static _model = "grok-beta";
+  constructor() {
+    super();
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)